web/templates/left_sidebar.hbs

```diff
@@ -64,10 +64,13 @@
                         <i class="fa fa-calendar" aria-hidden="true"></i>
                     </span>
                     {{~!-- squash whitespace --~}}
-                    <span>{{t 'Scheduled messages' }}</span>
+                    <span class="top_left_texts">{{t 'Scheduled messages' }}</span>
```

Comment:
This class name seems poor too; we're trying to get rid of the `top_left` class of name. `left-sidebar-navigation-label` might be OK.